packaging/nnstreamer.spec

```diff
@@ -302,6 +305,10 @@ BuildRequires:	npu-engine-devel
 BuildRequires: onnxruntime-devel
 %endif
 
+%if 0%{?ncnn_support}
+BuildRequires: ncnn-devel
```

Comment:
Let's force-enable ncnn build option if ncnn_support is 1. This prevents regression.

Reply:
@myungjoo Updated. PTAL.